test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageDependencyGroupTests.cs

```diff
@@ -218,5 +222,108 @@ public void PackageDependencyGroup_GetHashCode_SameVersionAndPackages_DifferentO
 
             Assert.Equal(aHashCode, bHashCode);
         }
+
+        [Fact]
+        public void PackageDependencyGroup_Deserialize_ReturnsExpected()
+        {
+            // Arrange
+            const string id = "PackageA";
+            var version = new VersionRange(new NuGetVersion(1, 4, 1));
+            var expectedPackageDependency = new PackageDependency(id, version);
+            IEnumerable<PackageDependency> packages = new List<PackageDependency>() { expectedPackageDependency };
+
+            var targetFramework = NuGetFramework.Parse(".NETStandard1.0");
+
+            // Act
+            using var stringReader = new StringReader(PackageRegistrationDependencyGroupsJson);
+            using var jsonReader = new JsonTextReader(stringReader);
+            jsonReader.Read();
+
+            var serializer = JsonSerializer.Create(JsonExtensions.ObjectSerializationSettings);
+            PackageDependencyGroup actualPackageDependencies = serializer.Deserialize<PackageDependencyGroup>(jsonReader);
+
+            // Assert
+            Assert.Equal(packages, actualPackageDependencies.Packages);
```

Comment:
```suggestion
            Assert.Equal(expectedPackages, actualPackageDependencies.Packages);
```

nitpick: small naming changes would help improve readability